package.json

```diff
@@ -99,7 +99,7 @@
     "@hot-loader/react-dom": "^16.13",
     "@reach/auto-id": "^0.13.0",
     "@reach/combobox": "^0.12.1",
-    "@reach/menu-button": "0.7.4",
+    "@reach/menu-button": "0.8.6",
     "@reach/rect": "^0.16.0",
```

Comment:
In Notifications Page, hovering over a notification does not reveal the 3-dot context menu -- one would need to hover on that specific area before it visible.  I was on Light Theme.
Comments seem to work correctly -- hovering anywhere would reveal the button.

Reply:
I've already fixed that yesterday `(25ed843).` . The issue should no longer exist.

Reply:
Fixed.
![image](https://user-images.githubusercontent.com/94839278/150327434-77ce8be1-8bbf-4b8f-ad94-c74d34dfddfd.png)